checks/core/resource_age.py

```diff
@@ -0,0 +1,26 @@
+"""
+The age of the specified resource at URL should be less than the specified maximum.
+
+The age in hours is returned.
+"""
+
+from telescope.typings import CheckResult
+from telescope.utils import fetch_head, utcfromhttpdate, utcnow
+
+
+EXPOSED_PARAMETERS = ["url", "max_age_hours"]
+DEFAULT_PLOT = ".age_hours"
+
+
+async def run(url: str, max_age_hours: int) -> CheckResult:
+    _, headers = await fetch_head(url)
+
+    last_modified = headers.get("Last-Modified", "Mon, 01 Jan 1970 00:00:00 GMT")
```

Comment:
Can the `last_modified` date header actually be empty or non-existent? If it should never be empty maybe it's worth throwing an error?